recipes/juypterlab-language-packs/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "jupyterlab-language-packs" %}
+{% set version = "0.0.2.dev0" %}
+{% set build_number = 0 %}
+{% set locales = ["ar-SA", "no-NO"] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: ar-SA
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name[-1] }}-ar-SA/{{ name[-1] }}-ar-SA-{{ version }}.tar.gz
+    sha256: d0e4a6d9ceeee573761e34731a6426a193956f7102ddfd40ae04620edb3dd61d
+  - folder: no-NO
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name[-1] }}-no-NO/{{ name[-1] }}-no-NO-{{ version }}.tar.gz
+    sha256: 3f1bcaa8d8c40d5e1cb3051a78c584203ade87286cff492fc19878dc1027e9f5
+
+build:
+  number: {{ build_number }}
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  commands:
+    - echo "no metapackage"
+
+outputs:
+{% for locale in locales %}
```

Comment:
This is going to mess a lot with automation. An alternative would be to list the locals in `conda_build_config.yaml`. That will work better with the bots but produce a really large CI matrix.

Reply:
Thanks for the early feedback @xhochy 
The for-loop is not working - locally it ends up with no outputs in the recipe. I'll give it a try with `conda_build_config`.